### DISCUSSION TITLE
Making bootable USB from ISO

### DISCUSSION THREAD

**Discussion by jpowellmlo** [PartialVolume/shredos.x86_64#222] [Q&A]
I am trying to use Rufus to make a bootable ISO out of the latest version (2023.08.2.25.0) and have tried both the i586 and x86_x64 versions.  Rufus tells me the ISO is either non-bootable or uses a boot or compression method not supported by Rufus.  I've tried multiple versions of Rufus and have confirmed with a different ISO that Rufus is working.  I also tried the previous release without luck.  I do notice that the current release's ISO is ~219MB whereas the previous version is ~73MB.  very large difference.  How do I go about making a bootable iso and why the large size difference between the two versions?

**Comment by jpowellmlo** [comment]
The .img files are accepted properly by Rufus.

**Comment by PartialVolume** [comment]
I've tried those .isos on Ventoy and burned onto CD so I know they are bootable. However, unlike the .imgs those isos are only legacy bootable, there is no EFI code, however I wouldn't have thought that mattered to Rufus.

Does Rufus have some sort of verbose mode that might be a bit more exact about what the problem is? In buildroot for an iso build I have to specify the boot device as cd, for the .img it has to be hd0, msdos1 so grub knows where it's supposed to be booting from. I don't know if Refus handles that appropriately or whether it's something else altogether. Hence why a more verbose error from Rufus might help figure it out.

**Comment by PartialVolume** [comment]
Maybe I need to apply isohybrid https://wiki.syslinux.org/wiki/index.php?title=Isohybrid to make it bootable on both CD/DVD and USB.

The size difference is due to the introduction of the Linux Direct Rendering Manager (DRM) and associated DRM graphics drivers that add over 100MB to the kernel. This has been introduced as we need to be able to send the device to sleep and wake it up after 5 seconds to unfreeze drives for secure erase. Without the DRM, the graphics system doesn't wake up reliably for all graphics cards and DRM handles that wake-up quite well most of the time.

**Comment by PartialVolume** [comment]
In buildroot it looks like it would be necessary to switch from grub2 to isolinux to use the isohybrid option.

I'll look into it and let you know.

**Comment by jpowellmlo** [comment]
The error message appears in Rufus upon selecting the ISO.  After hitting OK on the error message, I get returned to the Rufus main screen and the Boot selection box returns to "Disk or ISO image (please select)"  It doesn't even get to a place where it would let me try to write the USB stick.  Therefore, any verbose setting won't help.  I did look in Rufus, and the only setting it has is about checking for updates, so I don't see any obvious verbose settings anyway.  I will try booting using the usb stick burned with the img file today.  Thanks for looking into this.

**Comment by jpowellmlo** [comment]
Also, thanks for clarifying the size difference.
